src/factory.md

```diff
@@ -16,8 +16,8 @@ However, you can't simply generate widgets from a `Vec`.
 
 This is where factories are really useful.
 Custom collection types like `FactoryVecDeque` allow you to work with collections of data almost as comfortable as if they were stored in a `Vec`.
-At the same time, factories allow you to automatically visualize the data with widgets.
-Additionally, factories are very efficient by reducing the amount of UI updates to a minimum.
+At the same time, factories allow automatically visualization of data with widgets.
```

Comment:
"automatically visualization of data with widgets." is not correct and I would rather keep the previous version.